tests/e2e/restart_test.go

```diff
@@ -89,7 +91,7 @@ func (s *RestartTestData) TearDownSuite(c *C) {
 		c.Assert(err, IsNil, Commentf(out))
 
 		// Remove access for second datastore from the default vmgroup
-		out, err = adminutil.RemoveDatastoreFromVmgroup(s.config.EsxHost, adminconst.DefaultVMgroup, s.config.Datastores[1])
+		out, err = adminutil.RemoveDatastoreFromVmgroup(s.config.EsxHost, adminconst.DefaultVMgroup, "'"+ds2+"'")
```

Comment:
Yes, we remove it in SetUpSuite as we do strings.Compare() there and since config.Datastores has names with quotes whereas status["datastore"] has names without quotes, comparison fails.
```
if strings.Compare(status["datastore"], strings.Replace(s.config.Datastores[0], "'", "", -1)) == 0 {
		ds2 = s.config.Datastores[1]
	}
```